source/develop/release-management/features/virt/engine-dedicated-cpus.md

```diff
@@ -143,5 +138,28 @@ User wants to start VM B that requires 2 vCPUs and has dedicated cpu policy
 ## Limitations
 - Should only be supported with 4.7 cluster level
 
-## TBD
-- Siblings and isolate-threads
\ No newline at end of file
+
+
+### Isolate threads/Siblings
+
+Since we already know the available host capabilities and the available CPUs
+
+in case a VM is set with Isolate threads or sibilings, we can remove the whole core from the available cpu data structure
+
+Example:
+
+Host A with 1 Socket, 2 Cores in socket, 2 CPUs in each core
+
+hostToAvailableCpu data structure:
+
+{Host A id} -> {socket id} -> {core 1 id} -> {1,2}
+{Host A id} -> {socket id} -> {core 2 id} -> {3,4}
+
+Lets assume we have 1 VM with 2 vCPUs with isolate threads policy
+
+the VM will use from cpu ids 1 and 3, and cpu ids 2 and 4 will be blocked
+
+hostToAvailableCpu data structure after this change will be empty, since there is no available shared CPUs 
+current CPU pinning for the VM will be 1#1 2#3
```

Comment:
If you really wish to be on point: 0#1, 1#3.
We count from 0, but as it is only example and the host example starting from 1, that's OK as well.